### ISSUE TITLE
Curve Deformer in Sverchok

### ISSUE THREAD

**Issue by enzyme69** [nortikin/sverchok#1181]
If not using Curve Deformer, I know this is possible in Sverchok, what would be the sane way to do this? 

![screen shot 2017-01-27 at 12 24 27](https://cloud.githubusercontent.com/assets/3952687/22357379/aabccc82-e48b-11e6-82ff-7c2b7969263a.png)

https://gist.github.com/6a2689d97f9c872b99d411aa131d14ca

**Comment by enzyme69** [comment]
Actually.... maybe transform matrix and rotating while move the along "path" can do this. One sec...

**Comment by enzyme69** [comment]
![screen shot 2017-01-27 at 12 41 47](https://cloud.githubusercontent.com/assets/3952687/22357700/fff273e4-e48d-11e6-8fd4-20434c9737cb.png)

Twisted Torus... but only 1 rotation

**Comment by enzyme69** [comment]
https://gist.github.com/dca8697cbc062eba0482b61a449cf56b

I didn't know about this.

![screen shot 2017-01-27 at 12 47 47](https://cloud.githubusercontent.com/assets/3952687/22357820/d5a09610-e48e-11e6-8a92-baf5d24eea74.png)

**Comment by enzyme69** [comment]
Can be simpler, I think I made it more complicated than it has to.

https://gist.github.com/812ee9bfbc6be4840bd78ae14e623104

**Comment by enzyme69** [comment]
https://gist.github.com/3210b9c4b0503dd51236ed3a2101596c

I actually made this because I was looking at this that was posted at Blender FB Pages.

I finally understand the matter. And the Array and Curve Modifier thing is indeed something that Blender Artist often use.

![spiral_matter](https://cloud.githubusercontent.com/assets/3952687/22358712/4990b2ca-e495-11e6-8e1a-4163f2376540.jpg)

Since we don't have Curve Modifier yet, and I kind of confused matter a bit.

I decided to go back to original recipe (that low res image) and figure it out.
